src/Storage/custom/hybrid-2019-03-01/Update-AzStorageAccount.ps1

```diff
@@ -0,0 +1,373 @@
+        [System.Management.Automation.SwitchParameter]
+        # Allows https traffic only to storage service if sets to true.
+        ${EnableHttpsTrafficOnly},
+    
+        #[Parameter(ParameterSetName='UpdateExpanded', Mandatory, HelpMessage='The encryption keySource (provider). Possible values (case-insensitive): Microsoft.Storage, Microsoft.Keyvault')]
+        #[Parameter(ParameterSetName='UpdateViaIdentityExpanded', Mandatory, HelpMessage='The encryption keySource (provider). Possible values (case-insensitive): Microsoft.Storage, Microsoft.Keyvault')]
+        #[ArgumentCompleter([Microsoft.Azure.PowerShell.Cmdlets.Storage.Support.KeySource])]
+        #[Microsoft.Azure.PowerShell.Cmdlets.Storage.Category('Body')]
+        #[Microsoft.Azure.PowerShell.Cmdlets.Storage.Support.KeySource]
+        # The encryption keySource (provider). Possible values (case-insensitive): Microsoft.Storage, Microsoft.Keyvault
+        #${EncryptionKeySource},
+        
+        [Parameter(ParameterSetName='UpdateExpanded')]
+        [Parameter(ParameterSetName='UpdateViaIdentityExpanded')]
+        [System.Management.Automation.SwitchParameter]
+        ${NoEncryption},
+
+        [Parameter(ParameterSetName='UpdateExpandedStorageEncryption')]
+        [Parameter(ParameterSetName='UpdateViaIdentityExpandedStorageEncryption')]
+        [System.Management.Automation.SwitchParameter]
+        ${StorageEncryption},
+
+        [Parameter(ParameterSetName='UpdateExpandedKeyVaultEncryption')]
+        [Parameter(ParameterSetName='UpdateViaIdentityExpandedKeyVaultEncryption')]
+        [System.Management.Automation.SwitchParameter]
+        ${KeyVaultEncryption},
+    
+        [Parameter(ParameterSetName='UpdateExpanded', HelpMessage='A boolean indicating whether or not the service encrypts the data as it is stored.')]
+        [Parameter(ParameterSetName='UpdateExpandedStorageEncryption', HelpMessage='A boolean indicating whether or not the service encrypts the data as it is stored.')]
+        [Parameter(ParameterSetName='UpdateExpandedKeyVaultEncryption', HelpMessage='A boolean indicating whether or not the service encrypts the data as it is stored.')]
+        [Parameter(ParameterSetName='UpdateViaIdentityExpanded', HelpMessage='A boolean indicating whether or not the service encrypts the data as it is stored.')]
+        [Parameter(ParameterSetName='UpdateViaIdentityExpandedStorageEncryption', HelpMessage='A boolean indicating whether or not the service encrypts the data as it is stored.')]
+        [Parameter(ParameterSetName='UpdateViaIdentityExpandedKeyVaultEncryption', HelpMessage='A boolean indicating whether or not the service encrypts the data as it is stored.')]
+        [Microsoft.Azure.PowerShell.Cmdlets.Storage.Category('Body')]
+        [System.Management.Automation.SwitchParameter]
+        # A boolean indicating whether or not the service encrypts the data as it is stored.
+        ${FileEnabled},
```

Comment:
`EncryptFileService`  This should only be in parameter sets that include encryption, right?

Reply:
In the original cmdlets these "Enabled" encryptions are never assigned. I will try to test those.